web/template/watch.gohtml

```diff
@@ -268,18 +268,31 @@
                         x-init="watch.seekbarHighlights.init('{{$stream.ID}}');"
                 >
                     <template x-for="marker in seekbarMarker">
-                        <div class="absolute bg-gradient-to-t from-black bottom-0 top-0 w-[2px] rounded-t" :style="{ left: `${marker.position * 100}%` }"></div>
+                        <div class="absolute bottom-0 top-0" :style="{ left: `${marker.position * 100}%` }">
+                            <!-- Section Separator -->
+                            <div x-show="marker.type === watch.MarkerType.sectionSep" class="w-[2px] h-full bg-gradient-to-t from-black rounded-t"></div>
+
+                            <!-- Bookmark Marker -->
+                            <div x-show="marker.type === watch.MarkerType.bookmark" class="block w-[6px] h-[6px] ml-[-3px] rounded bg-indigo-400 rounded-t"></div>
+                            <div
+                                x-show="marker.type === watch.MarkerType.bookmark && hoverPos.onTarget(marker.position, 7)"
+                                class="absolute z-10 top-[-45px] text-center w-[100px] bg-indigo-400 ml-[-50px] px-[12px] py-2 rounded"
```

Comment:
```suggestion
                                class="break-all absolute z-10 top-[-45px] text-center w-[100px] bg-indigo-400 ml-[-50px] px-[12px] py-2 rounded"
```
Long words might break out of the boundaries of the box that is shown on hover. This should fix this.